Modules/_testcapi/object.c

```diff
@@ -131,13 +131,22 @@ pyobject_enable_deferred_refcount(PyObject *self, PyObject *obj)
     return PyLong_FromLong(result);
 }
 
+
+static PyObject *
+is_immortal(PyObject *self, PyObject *op)
```

Comment:
Oh, I didn't know that `Modules/_testcapi/immortal.c` exists. Maybe put the new function there?